tests/Auth0.ManagementApi.IntegrationTests/LogStreamsTests.cs

```diff
@@ -0,0 +1,126 @@
+﻿using Auth0.Core.Exceptions;
+using Auth0.ManagementApi.Models;
+using Auth0.Tests.Shared;
+using FluentAssertions;
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace Auth0.ManagementApi.IntegrationTests
+{
+    public class LogStreamsTests : TestBase, IAsyncLifetime
+    {
+        private ManagementApiClient _apiClient;
+
+        public async Task InitializeAsync()
+        {
+            string token = await GenerateBruckeManagementApiToken();
+            _apiClient = new ManagementApiClient(token, GetVariable("BRUCKE_MANAGEMENT_API_URL"));
+        }
+
+        public Task DisposeAsync()
+        {
+            _apiClient.Dispose();
+            return Task.CompletedTask;
+        }
+
+        [Fact]
+        public async Task Test_log_stream_crud_sequence()
+        {
+            // Create a new entity
+            var name = "Auth0.net Test Log Stream";
+
+            var request = new LogStreamCreateRequest
+            {
+                Name = name,
+                Type = LogStreamType.Http,
+                Sink = new
+                {
+                    httpEndpoint = "https://my-stream.com",
+                    httpContentType = "application/json",
+                    httpContentFormat = "JSONOBJECT",
+                    httpAuthorization = "http-auth"
+                }
+            };
+
+            var createdLogStream = await _apiClient.LogStreams.CreateAsync(request);
+            createdLogStream.Should().NotBeNull();
+            createdLogStream.Name.Should().Be(name);
+
+            // Get an entity
+            var fetchedLogStream = await _apiClient.LogStreams.GetAsync(createdLogStream.Id);
+            fetchedLogStream.Should().NotBeNull();
+            fetchedLogStream.Name.Should().Be(name);
+            fetchedLogStream.Id.Should().Be(createdLogStream.Id);
+
+            // Update the entity
+            var updateRequest = new LogStreamUpdateRequest
+            {
+                Name = "Auth0.net Test Updated Stream",
```

Comment:
Set the status to paused here, just in case it doesn't get deleted, so the Auth0 server doesn't try and send data to the (fake) endpoint.